# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1265150843 18000
#      Tue Feb 02 17:47:23 2010 -0500
# Node ID 14eba566f9f026d0dea5e99385cae0c63787fea3
# Parent  85d76f68923655f01e41160929198862b124c53e
use DEFUNX instead of DEFUN for canonicalize_file_name

diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,8 +1,12 @@
+2010-02-02  John W. Eaton  <jwe@octave.org>
+
+	* syscalls.cc (Fcanonicalize_file_name): Use DEFUNX instead of DEFUN.
+
 2010-02-02  Thomas Weber  <thomas.weber.mail@gmail.com>
 
 	* DLD-FUNCTIONS/filter.cc: New tests.
 
 2010-02-02  John W. Eaton  <jwe@octave.org>
 
 	* octave.cc (octave_main): Assume we have atexit.
 	* sighandlers.h, sighandlers.cc: Delete conditional definition
diff --git a/src/syscalls.cc b/src/syscalls.cc
--- a/src/syscalls.cc
+++ b/src/syscalls.cc
@@ -1593,17 +1593,17 @@ child process was resumed by delivery of
     }
 #else
   warning ("WIFCONTINUED always returns false in this version of Octave");
 #endif
 
   return retval;
 }
 
-DEFUN (canonicalize_file_name, args, ,
+DEFUNX ("canonicalize_file_name", Fcanonicalize_file_name, args, ,
   "-*- texinfo -*-\n\
 @deftypefn {Built-in Function} {[@var{cname}, @var{status}, @var{msg}]} canonicalize_file_name (@var{name})\n\
 Return the canonical name of file @var{name}.\n\
 @end deftypefn")
 {
   octave_value_list retval;
 
   if (args.length () == 1)
